### ISSUE TITLE
Trigger pull_request_review_comment on the prow-commands workflow

### ISSUE THREAD

**Issue by hermitkim1** [cloud-barista/cb-tumblebug#401]
**What would you like to be enhanced**
: I'd like to **trigger pull_request_review_comment** on the prow-commands workflow.

**Why is this needed**
: The prow-commands workflow runs by triggering `pull_request_review_comment`.

When I leave a comment on GitHub review interface, prow-commands may not work.

**Proposed solution**
[Proposal]
```yaml
on:
  issue_comment:
    types: [ created, edited ]
  pull_request_review_comment: 
    types: [ created, edited ]
```

[AS-IS]
```yaml
on:
  issue_comment:
    types: [created]
```

**Comment by seokho-son** [comment]
@hermitkim1 그렇군요! 추가하면 좋을 것 같습니다! ^^

**Comment by hermitkim1** [comment]
@seokho-son @jihoon-seo 

GitHub docs를 보다보니 `edited`도 추가할 필요가 있을 것 같아 Issue 내용 수정하였습니다.

**Comment by seokho-son** [comment]
Resolved by #402
Thank you !
